advocacy_docs/repos/working_with_repos/integrating/artifactory.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Using Hosted Artifactory with EDB repositories
+navTitle: Hosted Artifactory
+description: How to use Hosted JFrog Artifactory with EDB repositories
+---
+
+To set up Hosted JFrog Artifactory with EDB repositories, you need to create remote repositories in Artifactory. This article shows how to use Hosted JFrog Artifactory with EDB repositories. The process is different for on-premises Artifactory.
+
+## Get the base URLs
+
+The process is requires a base URLs to access the EDB repositories.
+
+1. From your EDB account, [get your token](../../getting_started/with_cli/get_your_token).
+
+1. Make a note of your token, for our examples, the token will be `xyz123`.
+
+1. Select the repository you want to access, for example, `standard`. :
+
+1. Your base URL will is then formed as:
+
+    ```
+    https://downloads.enterprisedb.com/<token>/<repository>/
+    ```
+
+    For the `standard` with our token, the repository, the base URL will be:
+
+    ```
+    https://downloads.enterprisedb.com/xyz123/standard/
+    ```
+
+1. Append the appropriate path for the repository type. 
+   For example, for a Red Hat Enterprise Linux 9 on x86 architecture, append `rpm/el/9/x86_64/` and seperately append `rpm/el/9/noarch/` for the noarch packages, to create two base URLs. With our example, the base URLs will be:
+
+    ```
+    https://downloads.enterprisedb.com/xyz123/standard/rpm/el/9/x86_64/
+    https://downloads.enterprisedb.com/xyz123/standard/rpm/el/9/noarch/
+    ```
+
+## Setup Artifactory remote repositories
+
+With your base URLs, you can now create the remote repositories in Artifactory.
+
+Use the Artifactory UI to create the 2 repositories (`x86_64` and `noarch`), one for each base URL. The `x86_64` repository will contain the `rpm` packages specifically for the `x86_64` architecture, and the `noarch` repository will contain the `rpm` packages which are for any architecture.
+
+Starting with the `x86_64` base URL:
+
+1. Select **Create Repository** and choose **Remote**.
+1. Choose **rpm** as the repository type.
+1. Enter a unique name for the repository key, for example, `edb-x86_64`.
+1. Fill in the URL as `https://downloads.enterprisedb.com/xyz123/standard/rpm/el/9/x86_64/`.
+1. Select **Create Remote Repository**.
+
+At this point, a prompt to setup the `yum` client configuration appears. This client configuration is the configuration you use on your local server or VM to access the configured EDB repositories via Artifactory.
+
+6.  On your local RHEL 9 server or VM create the yum repository file `/etc/yum.repos.d/artifactory-edb.repo`.
+    
+7.  Add the text generated by Artifactory to the `/etc/yum.repos.d/artifactory-edb.repo` file. The generated text should resemble:
+    
+    ```
+    [Artifactory]
+    name=Artifactory
```

Comment:
Perhaps this could be `artifactory_enterprisedb-standard`, so if they copy/paste it doesn't conflict with other examples they may have followed.

Reply:
I think this piece of code is generated directly by Artifactory after `Create Remote Repository`; in fact, some suggested modifications are provided afterward.